bookserver/session.py

```diff
@@ -30,6 +30,7 @@ async def load_user(user_id: str) -> schemas.User:
     rslogger.debug(f"Going to fetch {user_id}")
     user = await fetch_user(user_id)
     if user:
+        # TODO: I don't understand -- why do this here? Are we validating an existing user object?
```

Comment:
I don't see what benefit we get from separation; the db data has already been validated / put in a known format, so that seems like what we need with additional steps.

Reply:
One benefit we do get is that a Pydantic object is mypy-compatible as well as supporting autocomplete in some editors.  Database results remain opaque.  But that may not be enough of a benefit.

Reply:
I need to read up more on how FastAPI integrates with sqlalchemy. I'd also like to see an example of validating HTTP parameters, then storing that in the db, to see how the code ends up looking. That will help shape the "right" way to do things.

Reply:
in `routers/rslogging.py` the `log_book_event` does just that.  As well as `routers/assessment.py` and the `get_assessment_results`

Reply:
Thanks, I'll take a look.

Reply:
I like the way these schemas work -- FastAPI is really nice. You did a great job finding a really solid framework!